Resources/Public/Css/Main.css

```diff
@@ -0,0 +1,69 @@
+@import url(x.css);
+@import url(y.css);
+
+/**
+ * Multi-line comment
+ */
+
+.selector-1,
+.selector-2,
+.selector-3[type="text"] {
+  background: linear-gradient(#fff, rgba(0, 0, 0, 0.8));
+  box-sizing: border-box;
+  display: block;
+  color: #333;
+}
+
+.selector-a,
+.selector-b:not(:first-child) {
+  padding: 10px !important;
+  top: calc(calc(1em * 2) / 3);
+}
+
+.selector-x { width: 10%; }
+.selector-y { width: 20%; }
+.selector-z { width: 30%; }
```

Comment:
Would it make sense to have some real CSS here, e.g., for the tea list items?

Reply:
ok, fine :)